server/sqlstore/playbook.go

```diff
@@ -48,7 +48,7 @@ func NewPlaybookStore(pluginAPI PluginAPIClient, log bot.Logger, sqlStore *SQLSt
 			"ConcatenatedInvitedUserIDs", "ConcatenatedInvitedGroupIDs", "InviteUsersEnabled",
 			"DefaultCommanderID", "DefaultCommanderEnabled",
 			"AnnouncementChannelID", "AnnouncementChannelEnabled",
-			"WebhookOnCreationURL", "WebhookOnCreationEnabled").
+			"WebhookOnCreationURL", "WebhookOnCreationEnabled", "WebhookOnStatusUpdateURL", "WebhookOnStatusUpdateEnabled").
```

Comment:
Can you move this to another line?
```suggestion
			"WebhookOnCreationURL", "WebhookOnCreationEnabled",
			"WebhookOnStatusUpdateURL", "WebhookOnStatusUpdateEnabled").
```